src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,353 @@
+        "grant_type": "client_credentials",
+        "scope": "add_preferences",
+        "client_id": "client_id_first_discovery",
+        "client_secret": "client_secret_first_discovery"
+    }
+}
+```
+
+#### GPII Preferences Server Specific Configuration ####
+
+When connecting to the the GPII Preferences server the First Discovery Server must be configured with the following specific values:
+
+<table>
+    <thead>
+        <tr>
+            <th>Configuration</th>
+            <th>Value</th>
+        <tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>
+                <p>
+                    Environment Variable:
+                    <code>GPII_OAUTH2_ACCESS_TOKEN_PATH</code>
+                </p>
+
+                <p>
+                    JSON Config Path:
+                    <code>securityServer.paths.token</code>
+                </p>
+            </td>
+            <td>
+                "/access_token"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                    Environment Variable:
+                    <code>GPII_OAUTH2_ADD_PREFERENCES_PATH</code>
+                </p>
+
+                <p>
+                    JSON Config Path:
+                    <code>securityServer.paths.preferences</code>
+                </p>
+            </td>
+            <td>
+                "/add-preferences?view=%view"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                    Environment Variable:
+                    <code>GPII_OAUTH2_AUTH_GRANT_TYPE</code>
+                </p>
+
+                <p>
+                    JSON Config Path:
+                    <code>authentication.grant_type</code>
+                </p>
+            </td>
+            <td>
+                "client_credentials"
+            </td>
+        </tr>
+        <tr>
+            <td>
+                <p>
+                    Environment Variable:
+                    <code>GPII_OAUTH2_AUTH_SCOPE</code>
+                </p>
+
+                <p>
+                    JSON Config Path:
+                    <code>authentication.scope</code>
+                </p>
+            </td>
+            <td>
+                "add_preferences"
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+For more information about how the Preferences Server's security layer is configured. Please see the [Client Credentials Grant](https://wiki.gpii.net/w/GPII_OAuth_2_Guide#Client_Credentials_Grant) documentation.
+
+## REST API ##
+
+<table>
+    <thead>
+        <tr>
+            <th>URL</th>
+            <th>Request</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><code>/user?[view=:view]</code></td>
+            <td><code>POST</code></td>
+            <td>
+                Accepts a set of preferences, in a JSON object, to be stored on the preferences server. For example:
+
+                <code>
```

Comment:
The line breaks in all of these code/pre sections are not rendering as (I assume) you expect them to.